nav-kibana-dev.docnav.json

```diff
@@ -149,9 +140,6 @@
         {
           "id": "kibDevTutorialExpressions"
         },
-        {
-          "id": "kibDevDocsKPTTutorial"
-        },
```

Comment:
Some of the shared ux pages moved under "shared ux", but more important pages are still duplicated in the "tutorials" section for better discovery